src/d3a/models/market/two_sided_pay_as_bid.py

```diff
@@ -74,7 +74,7 @@ def bid(self, price: float, energy: float, buyer: str, seller: str, buyer_origin
             price = self._update_new_bid_price_with_fee(price, original_bid_price)
 
         bid = Bid(str(uuid.uuid4()) if bid_id is None else bid_id,
-                  price, energy, buyer, seller, original_bid_price, buyer_origin)
+                  self.now, price, energy, buyer, seller, original_bid_price, buyer_origin)
```

Comment:
I dont know whether this will be a problem, but i want to mention, that `self.now` is not the same as `pendulum.now()` . 
`self.now` is the time of the beginning current tick.

Reply:
Yes and thats what exactly is needing to plot offer/bid/trade simulated timestamp